cjdns-hdr/src/data_header.rs

```diff
@@ -128,10 +127,8 @@ mod tests {
     fn test_parse_unknown_content_type() {
         let hex_data = [
             // content type number out of IP6 range - 32000
-            "10007d00",
-            // content type number in IP6 range - 100
-            "10000064",
-            // content type out of available range (greater than 0x8000)
+            "10007d00", // content type number in IP6 range - 100
+            "10000064", // content type out of available range (greater than 0x8000)
```

Comment:
It seems that the formatting was good and now it is wrong. Why at all to change formatting in non-relevant places?